pg_rsupport.c

```diff
@@ -52,15 +52,15 @@ typedef struct saved_plan_desc
  * Functions used in R
  *****************************************************************************/
 void
-throw_pg_notice(const char **msg)
+throw_pg_log(int* elevel, const char **msg)
```

Comment:
I did not change the signature of R wrappers aside from returning invisible NULL now instead of a message. I do not think someone is using `.C` calls directly. I am sure it is safe to yank it out.